### PR TITLE
Usage docs, polish to the README/manifest/error feedback.

### PR DESCRIPTION
**Pull request by onelson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,10 +16,122 @@ Apparently *the blue crab* calls this sort of environment home.
...
+
+```
+$ cargo install --registry estuary my-cool-cli
+```
+
+Environment variables can also be used to configure cargo.
+See the docs on [using an alternate registry] and
+[publishing to an alternate registry] for more on this.
 
+[using an alternate registry]: https://doc.rust-lang.org/cargo/reference/registries.html#using-an-alternate-registry>
```
Comment (onelson) [inline README.md]:
```suggestion
[using an alternate registry]: https://doc.rust-lang.org/cargo/reference/registries.html#using-an-alternate-registry
```

File: README.md [hunk 2]
```diff
@@ -8,10 +16,122 @@ Apparently *the blue crab* calls this sort of environment home.
...
+estuary = { index = "http://estuary.example.com/git/index" }
+```
+
+With this entry added to your config, the next step is to "authenticate."
+
+```
+$ cargo login --registry estuary
+```
+
+> Note that Estuary currently does with the token to validate access.
```
Comment (WildMar) [inline README.md]:
```suggestion
> Note that Estuary currently does nothing with the token to validate access.
```

I'm assuming that was the intent, but there is a word missing in there

**Review by WildMar — CHANGES_REQUESTED** [review]
I haven't run through the instructions, but it makes sense to me (aside from the verbage on that one sentence)

File: README.md (hunk 2 above)
Comment (onelson) [reply README.md]: :smile: :cry: yep. Thanks!

**Pull request merged**